lms/djangoapps/instructor/views/api.py

```diff
@@ -509,8 +505,102 @@ def create_and_enroll_user(email, username, name, country, password, course_id):
     profile.country = country
     profile.save()
 
-    # try to enroll the user in this course
-    return CourseEnrollment.enroll(user, course_id)
+    return user
+
+
+def create_manual_course_enrollment(enrolled_user, course_id, mode, enrolled_by, reason, state_transition):
```

Comment:
Can we rename `enrolled_user` parameter to just `user`. this method does not expect an already enrolled user.